src/util/strip.js

```diff
@@ -1,5 +1,15 @@
 /** Strip HTML tags, convert nbsp to normal spaces, and trim. */
-export const strip = html => html
-  .replace(/<(?:.|\n)*?>/gm, '')
-  .replace(/&nbsp;/gm, ' ')
-  .trim()
+export const strip = (html, stripFormatting = true) => {
+  if (stripFormatting) {
+    return html
+    .replace(/<(?:.|\n)*?>/gm, '')
+    .replace(/&nbsp;/gm, ' ')
+    .trim()
+  }
+  else {
+    return html
+      .replace(/<(?:.|\n)*?([^bui])>/gm, '')
+      .replace(/&nbsp;/gm, ' ')
+      .trim()
+  }
```

Comment:
Notice the redundancy between these two sections of code. Can you think of a way to reuse the common portions, while using a ternary operator to switch on just the part that is different?